README.md

```diff
@@ -75,15 +86,45 @@ var express = require("express");
 var app = express();
 
 // Your code starts here...
+app.get('/authors', function(req, res) {
+  Author.find({}, function(authors) {
+    res.send('index')
+  })
+})
 
+app.post('/authors', function(req, res) {
+  Author.create({req.body}, function(author) {
+    res.send('create')
```

Comment:
You would probably want to send back a JSON representation of the author that was just created **or** redirect the user somewhere that makes sense (e.g., the show page for the newly created author)